### ISSUE TITLE
Cordova Plugins not available to Aurelia

### ISSUE THREAD

**Issue by jetstreamin** [jetstreamin/aurelia-cordova#1]
This application does not make the Cordova Plugins available to Aurelia. I am trying to work out how to import them currently. Once I get that done I will update the documentation for adding plugins to Cordova and how to make them available for consumption by Aurelia. If you know how to do this already please feel free to make a pull request and let's get that baked in.

**Comment by tomsean** [comment]
yes,i have use aurelia work in aurelia. and just use the bundle file.

**Comment by waynehaffenden** [comment]
Did you ever work out how to do this? Just seems to be no documentation or anything on how to access/use cordova and it's plugins within aurelia?
